integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/common/BridgeUtils.java

```diff
@@ -12,35 +12,31 @@
 
 public class BridgeUtils {
 
-    public static final String MANAGER_URL = Utils.getSystemProperty("event-bridge.manager.url");
-
-    protected static final String USER_NAME = Utils.getSystemProperty("bridge.token.username");
-    protected static final String PASSWORD = Utils.getSystemProperty("bridge.token.password");
-    protected static final String CLIENT_ID = Utils.getSystemProperty("bridge.client.id");
-    protected static final String CLIENT_SECRET = Utils.getSystemProperty("bridge.client.secret");
+    public static final String MANAGER_URL = System.getProperty("event-bridge.manager.url");
+    protected static final String CLIENT_ID = System.getProperty("bridge.client.id");
+    protected static final String CLIENT_SECRET = System.getProperty("bridge.client.secret");
 
     protected static String token;
```

Comment:
Nitpick - this can be removed too as token should be directly returned in `retrieveBridgeToken()`